app/src/main/java/ch/epfl/sweng/SDP/BaseActivity.java

```diff
@@ -1,18 +1,11 @@
 package ch.epfl.sweng.SDP;
 
-import android.os.Bundle;
-
 /**
  * Class to be inherited in activities where one wants to prevent the user from pressing the
  * back button.
  */
 public abstract class BaseActivity extends Activity {
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
```

Comment:
If you delete this method, you can delete the entire class, right?

Reply:
No, BaseActivity is needed for the onBackPressed()